wp-content/themes/core/pcss/components/_image.pcss

```diff
@@ -5,6 +5,7 @@
  * ----------------------------------------------------------------------------- */
 
 .c-image {
+	position: relative;
```

Comment:
What are you thinking this is for?

Reply:
we have a couple image component classes for different background image sizes (basically aspect ratio helpers) a little further down in this file. However, if the component (`c-image`) doesn't have position, the absolutely positioned background image will "bleed" to the next positioned container (by default the l-wrapper). Thus, I added a default position here so that the helper classes  can work out-of-the-box.  Happy to re-evaluate it. I don't know that I know of a solid system in place for how much is too much with default component styles.

Reply:
Ah right, I was thinking that bgd image class is added to this same el. Carry on.